examples/minimal-expo/metro.config.js

```diff
@@ -0,0 +1,47 @@
+const { getDefaultConfig } = require("expo/metro-config");
+const { FileStore } = require("metro-cache");
+const path = require("path");
+
+module.exports = withTurborepoManagedCache(
+  withMonorepoPaths(getDefaultConfig(__dirname)),
+);
+
+/**
+ * Add the monorepo paths to the Metro config.
+ * This allows Metro to resolve modules from the monorepo.
+ *
+ * @see https://docs.expo.dev/guides/monorepos/#modify-the-metro-config
+ * @param {import('expo/metro-config').MetroConfig} config
+ * @returns {import('expo/metro-config').MetroConfig}
+ */
+function withMonorepoPaths(config) {
+  const projectRoot = __dirname;
+  const workspaceRoot = path.resolve(projectRoot, "../..");
+
+  // #1 - Watch all files in the monorepo
+  config.watchFolders = [workspaceRoot];
+
+  // #2 - Resolve modules within the project's `node_modules` first, then all monorepo modules
+  config.resolver.nodeModulesPaths = [
+    path.resolve(projectRoot, "node_modules"),
+    path.resolve(workspaceRoot, "node_modules"),
+  ];
+
+  return config;
+}
+
+/**
+ * Move the Metro cache to the `node_modules/.cache/metro` folder.
+ * This repository configured Turborepo to use this cache location as well.
+ * If you have any environment variables, you can configure Turborepo to invalidate it when needed.
+ *
+ * @see https://turbo.build/repo/docs/reference/configuration#env
+ * @param {import('expo/metro-config').MetroConfig} config
+ * @returns {import('expo/metro-config').MetroConfig}
+ */
+function withTurborepoManagedCache(config) {
+  config.cacheStores = [
+    new FileStore({ root: path.join(__dirname, "node_modules/.cache/metro") }),
```

Comment:
Use Expo's FileStore to ensure CSS works. https://docs.expo.dev/versions/latest/config/metro/#tailwind-troubleshooting

```js
const { FileStore } = require('@expo/metro-config/file-store');
```

Reply:
Ehhm...

![CleanShot 2024-04-16 at 20 17 49](https://github.com/pingdotgg/uploadthing/assets/51714798/9d0dd5eb-13ab-498c-a084-cd65e04ab6de)

Reply:
ah needed to hoist the deps... classic pnpm x expo limitation